### PR TITLE
Rename master branch to main

### PR DESCRIPTION
**Pull request by jbonofre**
This PR rename resources using `master` to `main`. This PR should be merged only when the renaming has been done by Apache INRA.

### CODE REVIEW

**Review by nastra — APPROVED** [review]

File: README.md
Comment (jbonofre) [reply README.md:41]: It should if we update website repo. I will do another PR for website and docs. Let me check 😃

File: docs/flink-writes.md
Comment (ajantha-bhat) [inline docs/flink-writes.md:273]: nit: why these are shown as modified? Shall we revert the unrelated changed?

File: .github/workflows/spark-ci.yml
Comment (ajantha-bhat) [inline .github/workflows/spark-ci.yml:146]: nit: unrelated modification?

File: docs/metrics-reporting.md
Comment (ajantha-bhat) [inline docs/metrics-reporting.md:174]: same here.

File: docs/flink-writes.md
Comment (jbonofre) [reply docs/flink-writes.md:273]: Weird. Let me check and clean this.

Comment (jbonofre) [reply docs/flink-writes.md:273]: I used vim. It's maybe my vimrc :)  I gonna clean this ;)

File: .github/workflows/python-release.yml
Comment (rdblue) [inline .github/workflows/python-release.yml:54]: This is going to break CI? Is there a way one of the committers can commit this and change the main branch themselves to avoid affecting ongoing PRs?

**Comment by rdblue** [comment]
@jbonofre, how will this affect existing PRs? Do all of them need to be reopened against main instead of master?

**Comment by jbonofre** [comment]
@rdblue no impact on the existing PRs: they will be "attached" to main instead of master. The only impact is for the remotes for our users.

**Review by rdblue — APPROVED** [review]
Looks like this is good to go when the rename has happened.

**Comment by jbonofre** [comment]
Thanks, I do a cleanup on this PR and I create a INFRA ticket to actually rename.

File: .github/workflows/python-release.yml
Comment (jbonofre) [reply .github/workflows/python-release.yml]: I rebased to remove the python resources (as they are now on `iceberg-python` repo).

**Comment by jbonofre** [comment]
@ajantha-bhat I did a rebase to remove the python related resources, and the "formatting" should be OK now.

**Comment by jbonofre** [comment]
As soon as @ajantha-bhat vote +1 on this PR, I will create the ticket at Apache INFRA to trigger the rename.

**Comment by ajantha-bhat** [comment]
@jbonofre: LGTM. Thanks for fixing the formatting too.

**Comment by jbonofre** [comment]
`master` branch has been renamed to `main`. 

@Fokko @nastra if you can merge this PR when you have time, it would be great. Thanks !

**Comment by Fokko** [comment]
Thanks @jbonofre for taking the lead on this!

**Pull request merged**
